README.md

```diff
@@ -1,73 +1,93 @@
+<!-- @format -->
+
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Budget App
 
-> Description the project.
+> "Blog App" is a mobile web application where user can manage their budget from severals categories. In each categorie you can find a list of transactions with their corresponding amount in dollars($).
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Ruby
+- Rails
+- Postgres
+- Rspec
 
-## Live Demo (if available)
+## Getting Started
 
-[Live Demo Link](https://livedemo.com)
+## Live Demo
 
+[Heroku](https://gentle-garden-21828.herokuapp.com/)
```

Comment:
- Great job so far 👍🏿 Currently, a user cannot signup successfully using the deployed link on heroku, 
- [ ] kindly fix this, this is the error I see on signup 👍🏿  
![Screenshot_259](https://user-images.githubusercontent.com/69638013/174457743-dd436987-911a-4734-8026-7e96274527cf.png)